setup.py

```diff
@@ -13,13 +61,13 @@
 
         url='https://github.com/exaworks/psi-j-python',
 
-        classifiers=(
+        classifiers=[
             'Programming Language :: Python :: 3',
             'License :: OSI Approved :: MIT License',
-        ),
+        ],
 
 
-        packages=setuptools.find_packages(where='src'),
+        packages=['psi'],
```

Comment:
I think the previous `find_packages` is still required, as this only installs the top-level package:
```console
20s herbein1 ~/Repositories/exaworks/psi-j-python [.virtualenv] (remove_gcc_dependency ?)
❯ python -c 'from psi.j import Job'                                                                                                                                                                    9:12:11 ()
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'psi.j'
                                                                                                                                                                                                                  
(1) herbein1 ~/Repositories/exaworks/psi-j-python [.virtualenv] (remove_gcc_dependency ?)
❯ tree .virtualenv/lib/python3.9/site-packages/psi                                                                                                                                                     9:12:42 ()
.virtualenv/lib/python3.9/site-packages/psi
├── __init__.py
└── __pycache__
    └── __init__.cpython-39.pyc
```

With the `packages=find_packages(where='src')` code (which isn't quite the old code, since `find_packages` is directly imported, not sure how the old code worked actually...hmm):
```console
20s herbein1 ~/Repositories/exaworks/psi-j-python [.virtualenv] (remove_gcc_dependency !?)
❯ python -c 'from psi.j import Job'                                                                                                                                                                    9:13:39 ()
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/__init__.py", line 74, in <module>
    _find_plugins(path, type, 0)
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/__init__.py", line 63, in _find_plugins
    _find_plugins(path + os.path.sep + mod.name, type, ix + 1)
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/__init__.py", line 63, in _find_plugins
    _find_plugins(path + os.path.sep + mod.name, type, ix + 1)
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/__init__.py", line 63, in _find_plugins
    _find_plugins(path + os.path.sep + mod.name, type, ix + 1)
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/__init__.py", line 59, in _find_plugins
    _load_plugins(path, type)
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/__init__.py", line 47, in _load_plugins
    im = importlib.import_module('psi.j.{}.{}'.format(type.name, mod.name))
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/executors/__init__.py", line 4, in <module>
    from .rp import RPJobExecutor
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/executors/rp.py", line 17, in <module>
    class RPJobExecutor(JobExecutor):
  File "/Users/herbein1/Repositories/exaworks/psi-j-python/.virtualenv/lib/python3.9/site-packages/psi/j/executors/rp.py", line 124, in RPJobExecutor
    def _job_2_descr(self, job: Job) -> Dict[str, Any]:
NameError: name 'Dict' is not defined
                                                                                                                                                                                                                  
(1) herbein1 ~/Repositories/exaworks/psi-j-python [.virtualenv] (remove_gcc_dependency !?)
❯ tree .virtualenv/lib/python3.9/site-packages/psi                                                                                                                                                     9:13:44 ()
.virtualenv/lib/python3.9/site-packages/psi
├── __init__.py
├── __pycache__
│   └── __init__.cpython-39.pyc
└── j
    ├── __init__.py
    ├── __pycache__
    │   ├── __init__.cpython-39.pyc
    │   ├── exceptions.cpython-39.pyc
    │   ├── job.cpython-39.pyc
    │   ├── job_attributes.cpython-39.pyc
    │   ├── job_executor.cpython-39.pyc
    │   ├── job_executor_config.cpython-39.pyc
    │   ├── job_spec.cpython-39.pyc
    │   ├── job_state.cpython-39.pyc
    │   ├── job_status.cpython-39.pyc
    │   └── resource_spec.cpython-39.pyc
    ├── exceptions.py
    ├── executors
    │   ├── __init__.py
    │   ├── __pycache__
    │   │   ├── __init__.cpython-39.pyc
    │   │   ├── flux.cpython-39.pyc
    │   │   ├── local.cpython-39.pyc
    │   │   └── rp.cpython-39.pyc
    │   ├── flux.py
    │   ├── local.py
    │   └── rp.py
    ├── job.py
    ├── job_attributes.py
    ├── job_executor.py
    ├── job_executor_config.py
    ├── job_spec.py
    ├── job_state.py
    ├── job_status.py
    ├── launchers
    │   ├── __init__.py
    │   ├── __pycache__
    │   │   ├── __init__.cpython-39.pyc
    │   │   ├── launcher.cpython-39.pyc
    │   │   ├── mpirun.cpython-39.pyc
    │   │   ├── multiple.cpython-39.pyc
    │   │   ├── script_based_launcher.cpython-39.pyc
    │   │   └── single.cpython-39.pyc
    │   ├── launcher.py
    │   ├── mpirun.py
    │   ├── multiple.py
    │   ├── script_based_launcher.py
    │   └── single.py
    └── resource_spec.py

7 directories, 42 files
```

Reply:
Thank you for spotting this, sorry about that, and should be fixed now.